src/Microsoft.Health.Dicom.Functions/host.json

```diff
@@ -10,8 +10,8 @@
   },
   "DicomFunctions": {
     "Indexing": {
-      "BatchSize": 2,
-      "ParallelBatches": 2,
+      "BatchSize": 50,
```

Comment:
I'm afraid of exhausting resources for such a large batch.
If I understand correctly, this starts up to 50 threads , where each thread reindex on 1 DICOM instance.

Reply:
I have added a new property called `BatchThreadCount` to accomodate this